documentation/modules/rhv-prerequisites.adoc

```diff
@@ -8,6 +8,9 @@
 
 The following prerequisites apply to {rhv-full} migrations:
 
+* You must have one of the following:
+** {rhv-short} admin permissions. These permissions allow you to migrate any virtual machine in the system.
+** `DiskCreator` and `UserVmManager` permissions. These permissions have to be granted to you individually by an `admin` user per each individual virtual machine that you want to migrate.
```

Comment:
In addition to the individual permissions for VMs, any user who wants to add a new provider to the system must have at least the UserRole and ReadOnlyAdmin roles assigned to them. These are the minimum required permissions, however, any other administrative or superuser permissions will also work.

Reply:
@bkhizgiy Are you saying you need these roles in addition to `DiskCreator` and `UserVmManager` permissions or instead of them?